src/service/electionService.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import { ServerError , ApiError } from 'Errors';
+import ElectionRepo from '../repository/Election';
+import {ElectionManager}  from 'Managers';
+
+
+const getElectionById = async (req) => {
+    try {
+        const id = req.params.electionId;
+        const election = await ElectionRepo.getbyId(id);
+        if (!_.isEmpty(election)){
+            return ElectionManager.mapToElectionModel(election);
+        } else {
+            throw new ApiError("Election not found");
+        }
+    } catch (error) {
+        throw error;
```

Comment:
Throw the specific error with error code

Reply:
threw ServerError